_data/csp.js

```diff
@@ -37,7 +37,7 @@ const CSP = {
     ["object-src", quote("none")],
     // Script from same-origin and inline-hashes.
     // If you need to add an external host for scripts you need to add an item like 'https://code.jquery.com/jquery-3.6.0.slim.min.js' to this list.
-    ["script-src", SELF, /* Replaced by csp.js plugin */ "HASHES"],
+    ["script-src", SELF, /* Replaced by apply-csp.js plugin */ "HASHES", quote("unsafe-inline")],
```

Comment:
Was this intentional>

Reply:
Sorry, I restored it

Reply:
It was to avoid an warning with Lighthouse

Reply:
Could you paste the warning? I think this is probably a benign change, but it is super confusing, and the browsers that this excludes don't matter anymore.

Reply:
Sure.

"Consider adding 'unsafe-inline' (ignored by browsers supporting nonces/hashes) to be backward compatible with older browsers."

![image](https://user-images.githubusercontent.com/25171375/140078283-359a78d4-5e66-4b58-bead-5450fa4db994.png)